### PR TITLE
ci-operator/openshift/os: Remove C9S jobs from 4.16

### PR DESCRIPTION
**Pull request by travier**
Those test jobs only make sense on the main development branch.

### CODE REVIEW

**Review by HuijingHei — APPROVED** [review]

**Comment by LorbusChris** [comment]
OKD has two release streams: stable (== 4.16) and next (== 4.17). So I think these tests should still be run somewhere for 4.16

**Comment by travier** [comment]
> OKD has two release streams: stable (== 4.16) and next (== 4.17). So I think these tests should still be run somewhere for 4.16

SCOS should be built only once from the master branch and then specialized for OKD versions. It does not make sense to build SCOS based on older OCP.

/pj-rehearse skip

**Review by HuijingHei — APPROVED** [review]
/lgtm

File: ci-operator/jobs/openshift/os/openshift-os-release-4.16-presubmits.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 presubmits:
   openshift/os:
   - agent: kubernetes
-    always_run: false
+    always_run: true
```
Comment (c4rt0) [inline ci-operator/jobs/openshift/os/openshift-os-release-4.16-presubmits.yaml:4]: I'm just curious, what's the logic behind this boolean?

**Review by c4rt0 — APPROVED** [review]
lgtm

File: ci-operator/jobs/openshift/os/openshift-os-release-4.16-presubmits.yaml (hunk 1 above)
Comment (c4rt0) [reply ci-operator/jobs/openshift/os/openshift-os-release-4.16-presubmits.yaml:4]: Ok, NVM.
```    rerun_command: /test images```

Comment (travier) [reply ci-operator/jobs/openshift/os/openshift-os-release-4.16-presubmits.yaml:4]: This part is automatically generated so I never looked at it :)

**Comment by jlebon** [comment]
> > OKD has two release streams: stable (== 4.16) and next (== 4.17). So I think these tests should still be run somewhere for 4.16
> 
> SCOS should be built only once from the master branch and then specialized for OKD versions. It does not make sense to build SCOS based on older OCP.

I think the new `c9s` variant added in https://github.com/openshift/os/pull/1445 will make this really clear because you would then take this OCP/OKD-agnostic bootable container image and derive from it multiple times (once per supported OKD version). Those base composes don't consume the `c9s-sig-cloud-okd` repo.

**Pull request merged**